test/common/testvm.py

```diff
@@ -844,6 +844,44 @@ def virEventLoopNativeRun():
         self.eventLoopThread.setDaemon(True)
         self.eventLoopThread.start()
 
+TEST_DOMAIN_XML="""
```

Comment:
Why move this in here? I would be ok with renaming it to something `template`, but the idea was to not have a huge `testvm.py` file.